### ISSUE TITLE
Only allow specific google accounts through google oauth

### ISSUE THREAD

**Issue by ianshmean** [JuliaLang/JuliaBox#363]
Is it possible to use google oauth, but limit to specific google accounts?
I want the protection of oauth, but not for the system to be open to everyone.

**Comment by ViralBShah** [comment]
It's tough to do whitelist/blacklist in a general way that works for everyone. Ideally, it should be an approval workflow, I am guessing.

**Comment by ianshmean** [comment]
Agreed. For my purposes though I'm happy for it to be a static array of 2-3 email addresses, and I was wondering whether there's a field I could add to `/jboxengine/conf/jbox.user` to do that?

**Comment by tanmaykm** [comment]
Possible now with post authorization hooks provided in #381.

Post authorization plugin is missing a mention in [docs](https://github.com/JuliaLang/JuliaBox/blob/7605c724d14b0f2e1873c5df71f9c5829d052855/docs/Architecture.md) though.
